internal/pkg/state/view.go

```diff
@@ -282,25 +316,29 @@ func (v *View) MarketStorageDeal(ctx context.Context, dealID abi.DealID) (market
 
 func (v *View) MarketDealState(ctx context.Context, dealID abi.DealID) (*market.DealState, error) {
 	marketState, err := v.loadMarketActor(ctx)
-	dealStates := v.asDealStateArray(ctx, marketState.States)
+	if err != nil {
+		return nil, err
+	}
+
+	dealStates, err := v.asDealStateArray(ctx, marketState.States)
 	if err != nil {
 		return nil, err
 	}
 
 	return dealStates.Get(dealID)
 }
 
-// NetworkTotalPower Returns the storage power actor's value for network total power.
-func (v *View) NetworkTotalPower(ctx context.Context) (abi.StoragePower, error) {
+// NetworkTotalRawBytePower Returns the storage power actor's value for network total power.
```

Comment:
Power has been split into RawByte and QualityAdjusted power. It's not clear to me what the proper analogy to old Power is.